delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -28,6 +28,15 @@
 from openerp.tools.translate import _
 
 _compile_itemid = re.compile(r'[^0-9A-Za-z+\-_]')
+_logger = logging.getLogger(__name__)
+
+try:
```

Comment:
OK, it was told that way, although the constant warning each time the server is started is not optimal.